Remotion/ObjectBinding/Web/Res/HTML/BocAutoCompleteReferenceValue.jquery.js

```diff
@@ -1157,7 +1157,12 @@
                     continue;
                 var item = options.formatItem(data[i].data, i + 1, max, data[i].value, term);
                 var li = $ ("<li role='option' aria-selected='false' />")
-                  .html (options.highlight (item.html, term))
+                  .html (options.highlight (item.html, term
+                      .replace (/&/g, "&amp;")

```

Comment:
Not sure if this is the right place or if this should be in the highlight function

```javascript
highlight: function(value, term) {
    if (term == '')
      return value;
    return value.replace(new RegExp("(?![^&;]+;)(?!<[^<>]*)(" + term.replace(/([\^\$\(\)\[\]\{\}\*\.\+\?\|\\])/gi, "\\$1") + ")(?![^<>]*>)(?![^&;]+;)", "gi"), "<strong>$1</strong>");
},
```